src/Core/AcmeClientInterface.php

```diff
@@ -94,6 +94,24 @@ public function requestOrder(array $domains): CertificateOrder;
      */
     public function finalizeOrder(CertificateOrder $order, CertificateRequest $csr, int $timeout = 180, bool $returnAlternateCertificateIfAvailable = false): CertificateResponse;
 
+    /**
+     * Request authorization challenge data for a list of given domains.
+     *
+     * An AuthorizationChallenge is an association between a URI, a token and a payload.
+     * The Certificate Authority will create this challenge data and you will then have
+     * to expose the payload for the verification (see challengeAuthorization).
+     *
+     * @param array $domain the domains to challenge
+     *
+     * @throws AcmeCoreServerException        when the ACME server returns an error HTTP status code
+     *                                        (the exception will be more specific if detail is provided)
+     * @throws AcmeCoreClientException        when an error occurred during response parsing
+     * @throws ChallengeNotSupportedException when the HTTP challenge is not supported by the server
+     *
+     * @return AuthorizationChallenges[] the list of authorization challenge group data per domain as returned by the Certificate Authority
+     */
+    public function requestMultipleAuthorization(array $domains): array;
```

Comment:
Unfortunately, we can't add a method in an interface in minor releases. Instead, you should create a new `MultipleAuthorizationClientInterface` that extends `AcmeClientInterface`, that contains this method only and make the AcmeClient implement MultipleAuthorizationClientInterface.